### DISCUSSION TITLE
What does the -x parameter do?

### DISCUSSION THREAD

**Discussion by Muffexx** [Chia-Network/chia-blockchain#3798] [Q&A] [answered]
Hello there,

what does the -x parameter (plots create) do?
Maybe add a description to the [wiki](https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference).

regards
Muffexx

**Comment by ghost** [comment, marked as answer]
Does not add output directory to list of directories used for farming.
